resolver.go

```diff
@@ -61,6 +62,9 @@ func NewResolver(
 	computer *compute.Computer,
 	auth *auth.Authenticator,
 ) *Resolver {
+	defer sentry.Flush(2 * time.Second)
+	defer sentry.Recover()
```

Comment:
This is for handling panics in the main routine like this
https://sentry.io/organizations/dapperlabs/issues/3453939358/?project=6398442&query=&statsPeriod=14d